### DISCUSSION TITLE
Where are the images and files generated by Dify stored? Also, the tool's API cannot access the generated files.

### DISCUSSION THREAD

**Discussion by xm1085** [langgenius/dify#7380] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

![image](https://github.com/user-attachments/assets/e0226ad6-fbfb-4b4b-9ef0-83bb939c697b)
The generated files cannot be accessed or located.

**Comment by xiaoqingqy** [comment]
I am the same. I hope to obtain the URL at the next node so that I can download it, but the URL obtained at the next node is always empty
